libraries/libfc/include/fc/crypto/bls_common.hpp

```diff
@@ -4,25 +4,25 @@
 namespace fc::crypto::blslib {
 
    template <typename Container>
-   static Container serialize_base58(const std::string& data_str)
+   static Container serialize_base64(const std::string& data_str)
    {
 
       using wrapper = checksummed_data<Container>;
 
       wrapper wrapped;
 
-      auto bin = fc::from_base58(data_str);
+      auto bin = fc::base64_decode(data_str);
       fc::datastream<const char*> unpacker(bin.data(), bin.size());
       fc::raw::unpack(unpacker, wrapped);
-      FC_ASSERT(!unpacker.remaining(), "decoded base58 length too long");
+      FC_ASSERT(!unpacker.remaining(), "decoded base64 length too long");
       auto checksum = wrapper::calculate_checksum(wrapped.data, nullptr);
       FC_ASSERT(checksum == wrapped.check);
 
       return wrapped.data;
    }
 
    template <typename Container>
-   static std::string deserialize_base58( Container data, const yield_function_t& yield) {
+   static std::string deserialize_base64( Container data, const yield_function_t& yield) {

```

Comment:
You can remove all the `yield`s including in the function parameter.

Reply:
ah yes.